addons/l10n_cn/models/account_move.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    fapiao = fields.Char(string='Fapiao Number', size=8, copy=False, tracking=True)
+    in_china = fields.Boolean(compute='_compute_in_china', help='Technical field to compute if the company is in China.')
```

Comment:
I would like to make a change in generic accounting for that.  We should add 'country_code' related company_id.country_id.code in AccountMove and just have an attrs 'country_code', '=', 'CN'